frontend/src/components/browse-sightings-page/BrowseSightings.tsx

```diff
@@ -17,11 +20,23 @@ export const BrowseSightings: React.FC = () => {
   return (
     <>
       <h1>Reported Sightings</h1>
-      <ul>
-        {sightings.map((sighting, index) => (
-          <SightingCard sighting={sighting} key={sighting.id} />
-        ))}
-      </ul>
+      <button
+        onClick={() => {
+          setMapToggle(!mapToggle);
+        }}
+      >
+        Sightings Map
+      </button>
+      {!mapToggle ? (
+        <ul>
+          {sightings.map((sighting, index) => (
+            <SightingCard sighting={sighting} key={sighting.id} />
+          ))}
+        </ul>
```

Comment:
Split this into its own component, `SightingList.tsx`, that takes a list of sightings as a prop, so this becomes:
```suggestion
        <SightingList sightings={sightings} />
```